std/traits.d

```diff
@@ -2279,15 +2279,18 @@ usually used for compile-time reflection purposes.
 template FunctionTypeOf(func...)
 if (func.length == 1 && isCallable!func)
 {
-    static if (is(typeof(& func[0]) Fsym : Fsym*) && is(Fsym == function) || is(typeof(& func[0]) Fsym == delegate))
+    static if ((is(typeof(& func[0]) Fsym : Fsym*) && is(Fsym == function)) || is(typeof(& func[0]) Fsym == delegate))
```

Comment:
This change is unnecessary

Reply:
It is necessary for clarity. I based my change on DScanner suggestions. If you prefer I can revert it, but the rationale is that it clarifies the logic precedence of the expression.

https://github.com/dlang-community/D-Scanner/blob/2963358eb4a24064b0893493684d4075361297eb/src/dscanner/analysis/logic_precedence.d#L45